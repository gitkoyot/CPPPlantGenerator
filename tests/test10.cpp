#include "test10.h"


int glibal_function(char argument1, int argument2,test10_class* classT10)
{
	test10_class* ptrToTest10;
	return 0;
}

