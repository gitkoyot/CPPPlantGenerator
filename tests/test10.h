#ifndef TEST10_H
#define TEST10_H

class test10_class
{
public:
	test10_class();
	~test10_class();
private:
	/* data */
}; /* -----  end of class test10_class  ----- */


#endif
