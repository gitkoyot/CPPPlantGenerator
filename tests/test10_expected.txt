@startuml
class test10_class{
void test10_class ( )
void ~test10_class ( )
}
@enduml

